### PR TITLE
Implemented usage analytics

### PR DESCRIPTION
**Pull request by Wikijito7**
Created usage analytics to register metrics for ecibot-web.

Solves #12

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -184,6 +186,7 @@ async def sonidos(ctx):
 async def search(ctx, arg):
     blank_space = "\u2800"
     sounds_list = get_sound_list_filtered(arg)
+    database.register_user_interaction_async(ctx.author.global_name, "buscar")
```
Comment (javdc) [inline main.py]: For consistency it is better to put all names in English or always use the function name

Comment (javdc) [reply main.py]: And it would be better to use the ID or the user name instead of the alias

File: utils.py
Comment (javdc) [inline utils.py:1]: Check if the line break has been changed from LF to CRLF

File: main.py (hunk 1 above)
Comment (Wikijito7) [reply main.py]: Analytics naming is following the !help command naming.

image

**Pull request merged**